js/newtab.js

```diff
@@ -203,7 +206,7 @@ function getImage(info) {
 			}
 		};
 		// Actually open the request.
-		mediaReq.open('GET', 'https://www.thebluealliance.com/api/v3/team/frc' + teamInfo.num + '/media/' + new Date().getFullYear() + '?X-TBA-Auth-Key=IJ7ECNmOibpHt04EdVs4xS7q5OQkIY5GE7USErbLXK3i4obXAilhJD8VP590o8Ur');
+		mediaReq.open('GET', 'https://www.thebluealliance.com/api/v3/team/frc' + teamInfo.num + '/media/' + (month >= 4 ? new Date().getFullYear() : (new Date().getFullYear() - 1)) + '?X-TBA-Auth-Key=IJ7ECNmOibpHt04EdVs4xS7q5OQkIY5GE7USErbLXK3i4obXAilhJD8VP590o8Ur');
```

Comment:
You're creating two `Date` objects. You should create one, then in this line run `getMonth`.